data/microos/butane/script

```diff
@@ -57,6 +55,18 @@ EOF
 
 }
 
+test_prepare() {
+    echo "combustion: test_prepare function ran OK" > /dev/kmsg
```

Comment:
Wouldn't it be more robust to create a canary file that could also survive a reboot or a truncated journal?

e.g.

```suggestion
    echo "combustion: test_prepare function ran OK" > /root/combustion.canary
```